### ISSUE TITLE
File Uploads start but fail to complete after updating to version 2.30.0

### ISSUE THREAD

**Issue by danworsley** [filebrowser/filebrowser#3336]
As titled above, I updated my FileBrowser to v2.30.0 as it was quite out of date. I'm running it on Linux Lite 6.6 within a virtual machine on my Windows server. I've run filebrowser this way for quite some time and have various other VMs running on my server, such as HASSIO, MoodeAudio etc.

Filebrowser was working fine for me before, but since the update I can no longer seem to upload files. If I try to upload say a 100mb file, the upload process will start, and at a good upload speed, but after 5-10 seconds the upload speed starts to drop, and eventually drops to 0mbps. After that, I end up with a 0byte file in my folder. 

I have checked my CPU and RAM usage within the VM just to confirm it isn't a hardware throttling issue, CPU and RAM usage are normal throughout the upload (currently providing 2x Xeon Cores and 4GB ram to the VM, which is more than enough).

The data is stored on a USB 3.0 WD Portable Hard Drive which is dedicated to the VM. I've tested the drive and it appears to be fine. Again, I didn't have issues with it before the upgrade.

Also to note, I have enabled logging in my config.json file and set it to debug, no errors are being recorded.

Downloads work no problem, I tested it by downloading a 3.8GB file and I got a consistent speed and the download completed successfully. 

Has anyone experienced this issue? I can't find any similar posts on this issue, only that some people couldn't upload at all, but that's not the case for me as the uploads will start, they just never complete.

**Comment by danworsley** [comment]
Post update:

I've since tried using a different drive for root. I've created a virtual hard disk in virtualbox, transferred my files from the USB hard drive, updated systemd, config.json etc. to now use the new hard disk and removed the USB hard drive from the system.

I'm still getting the same results, so it doesn't appear to be an issue with the hard drive.

Another thing I've tried is adjusting the chunked uploads file size limit to 10GB and number of retries to 30.

This hasn't made a difference either.

I suspect the new Chunked Uploads feature might be the issue, as the old version of filebrowser I was using didn't have chunked upload support as far as I know.

I have tried adding the following line to my config.json:
"chunked_file_upload": false
Then restarted filebrowser, but it doesn't appear to have done anything.

Hoping to get a response soon!
Thanks

**Comment by n6v26r** [comment]
i also have this problem

**Comment by danworsley** [comment]
I think I've tried everything I can possibly think of, and given that Filebrowser hasn't seen any updates for quite some time, and issues posted seem to go unresolved, I've taken Filebrowser off my system. Such a shame really, it was a brilliant piece of software and I've been using it for many years. I've now deployed Filegator, a lot more basic but it does what I need it to do, and it's working perfectly.
